kn.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: kn
```

Comment:
Would the package name `knative-client` be better?

Reply:
Yes, org/repo naming convention is better. :)